distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
Not sure how I can make myself more clear, let me try again...

The user **is not building Way Cooler from source**
`distribute.sh` **is to be ran by us, we are building the binaries. It's automatically statically linked to wlc, the user doesn't even have to know this exists.**
We are offering **two** ways to install Way Cooler (excluding building from source):
1) The preferred way, they download tarred versions of Way Cooler. This includes:
  a) `way-cooler`
  b) `way-cooler-bg`
  c) `install.sh` <-- Which simply puts these binaries in their correct places
2) The least preferred way, but offered because I want to simplify the process for people to try out Way Cooler as much as humanly possible (and it will be noted as such) will be a `curl https://way-cooler.org/way-cooler-install.sh | sh`. This will run `way-cooler-install.sh`, which I **haven't written yet. It isn't one of these scripts**. 

We will not be offering the second option in the next release, only the first (and preferred) option.